catalyst-gateway/bin/src/service/api/cardano/staking/assets_get.rs

```diff
@@ -185,12 +183,18 @@ async fn get_txo_by_txn(
         let entry = txo_info
             .assets
             .entry(row.policy_id.clone())
-            .or_insert(TxoAssetInfo {
-                id: row.policy_id,
-                name: row.policy_name,
-                amount: num_bigint::BigInt::ZERO,
-            });
-        entry.amount += row.value;
+            .or_insert_with(Vec::new);
```

Comment:
Not quite get it this change.
Could you elaborate what we are doing here actually ?

Reply:
The main point is to make the value of the map to be able to store multiple assets for a policy id (previously one policy id for one asset only).